src/MPCC.cpp

```diff
@@ -282,8 +282,11 @@ void initialize(int &m, int &n, int &p, int seed,
 //This function is the implementation of a matrix x matrix algorithm which computes a matrix of PCC values
 //but increases the arithmetic intensity of the naive pairwise vector x vector correlation
 //A is matrix of X vectors and B is transposed matrix of Y vectors:
-//P = [ sum(AB) - (sumA)(sumB)/N] /
-//    sqrt[ ( sumA^2 -(1/N) (sum A/)^2)[ ( sumB^2 - (1/N)(sum B)^2) ]
+//P = [ sum(AB) - (sumA)(sumB)/N ] /
```

Comment:
Why was this part of the comment changed ? and what does it mean ? the commit message on the change doesn't mention why this change was needed and the comment is less readable now for any user coming after you

Reply:
This change fixes some brackets typos and adds an equation which is mutiplied by N in both numerator and denominator. If you see the code, it actually multiplies with N instead of dividing.